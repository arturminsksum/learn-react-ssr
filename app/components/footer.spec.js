import React from 'react';
import Footer from './footer';

describe('Footer', () => {
  it('should render Footer correctly', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
