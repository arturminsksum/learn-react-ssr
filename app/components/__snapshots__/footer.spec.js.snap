// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render Footer correctly 1`] = `
<footer
  className="footer"
>
  <div
    className="container"
  >
    <div
      className="content has-text-centered"
    >
      <p>
        <strong>
          ArchiNews
        </strong>
         by
        <a
          href=""
        >
          Artur Minsk
        </a>
        . The source code is licensed
        <a
          href="http://opensource.org/licenses/mit-license.php"
        >
          MIT
        </a>
        . The website content is provided by
        <a
          href="https://newsapi.org/"
        >
          News API
        </a>
        .
      </p>
    </div>
  </div>
</footer>
`;
